test/Test/Data/String/Base64.purs

```diff
@@ -65,14 +66,14 @@ testBase64 = do
   let
     encodeDecodeIdProp :: WellFormedInput -> Result
     encodeDecodeIdProp (WellFormedInput str) =
-      str === unsafePartial (fromRight <<< decode <<< encode $ str)
+      str === fromRight "err" (decode <<< encode $ str)
 
   quickCheck encodeDecodeIdProp
 
   log "decode <<< encodeUrl == id for well-formed input"
   let
     encodeUrlDecodeIdProp :: WellFormedInput -> Result
     encodeUrlDecodeIdProp (WellFormedInput str) =
-      str === unsafePartial (fromRight <<< decode <<< encodeUrl $ str)
+      str === fromRight "err" (decode <<< encodeUrl $ str)
 
-  quickCheck encodeUrlDecodeIdProp
+  quickCheck encodeUrlDecodeIdProp
```

Comment:
Newline is missing here :see_no_evil:

Reply:
> What a coincidence @CarstenKoenig meeting you here !
> We're using that libary in our projects and I just wanted to update it to purescript v0.14.0, looks like you were faster so 

it's a small world - especially the FP one I guess